stable/artifactory/templates/access-bootstrap-creds.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if not (and .Values.artifactory.accessAdmin.secret .Values.artifactory.accessAdmin.dataKey) }}
+{{- if .Values.artifactory.accessAdmin.password }}
```

Comment:
@eldada I actually think it’s good that we use the same method for when you provide a secret and when you don’t. This way you always know where it is stored instead of it being sometimes on disk and sometimes with a secret. WDYT?